javascript/javascript2/week2/hyfbay/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>HyfBay</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Barlow:wght@200&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/f44cd04f18.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+
+    <main>
+        <div id="searchItem">
+                <i class="fas fa-search"></i>
+                <input id="searchProduct" placeholder="Search by product">
```

Comment:
It is always a good practice to link input tags with related labels. You can read more here: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label